README.md

```diff
@@ -161,15 +161,17 @@ for each Pull Request, and on merge to `main`. You can run CI locally using a to
 like [act](https://github.com/nektos/act).
 
 ## HTTP Endpoints
-You can find all HTTP endpoints by checking out the swagger docs at: `http://localhost:8002/docs`
+You can find all HTTP endpoints by checking out the swagger docs at: `http://localhost:8080/swagger`
```

Comment:
Just tried this and it didn't work :(

Reply:
Here's what it looked like for me, after running `mage run`

![image](https://github.com/TBD54566975/ssi-service/assets/1139998/792dd9b6-8285-434f-be64-08ce872145a0)